portfolio.html

```diff
@@ -0,0 +1,231 @@
+        </div>
+        <div class="item1b">
+          <h2>Multi-Post Stories</h2>
+          <div class="frame2">
+           <span>FACEBOOK</span>
+            <ul class="role">
+              <li>Full Stack Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p >
+            Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+          </p>
+          <div class="tags">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+            <button type="button"><a href="">See Project</a></button>
+          </div>
+        </div>
+      </div>
+      <div class="item3">
+        <div class="item3a">
+          <img src="images/Snapshoot%20Portfolio3.png" alt="" />
+        </div>
+        <div class="item1b">
+          <h2 class="facebook">Facebook 360</h2>
+          <div class="frame2 facebook">
+            <span>FACEBOOK</span>
+            <ul class="role">
+              <li>Full Stack Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p class="facebook">
+            Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+          </p>
+          <div class="tags facebook">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+            <button type="button"><a href="">See Project</a></button>
+          </div>
+        </div>
+      </div>
+      <div class="item4">
+        <div class="item4a">
+          <img src="images/Snapshoot%20Portfolio4.png" alt="" />
+        </div>
+        <div class="item1b">
+          <h2>Uber Navigation</h2>
+          <div class="frame2">
+            <span>Uber</span>
+            <ul class="role">
+              <li>Lead Developer</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+           Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+          </p>
+          <div class="tags">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+            <button type="button"><a href="">See Project</a></button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="grid-container2">
+      <div class="about-me">
+        <h1>
+          About Myself
+        </h1>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
```

Comment:
- [ ] Please make sure your project match the figma design even when resized. Consider to also remove the second github icons and replace it by the appropriate icon.
**Your project**
![e4](https://user-images.githubusercontent.com/74411135/229213272-bd970260-815e-4fc8-b60d-5192c9bf3d99.png)

**Figma design**
![image](https://user-images.githubusercontent.com/74411135/229213346-7c5fbcfa-11c0-41fc-be97-c91220b9753d.png)

This will make your design prettier which will improve the user experience 📱 💻 🖥️ 😃 😺